pkg/maps/ctmap/gc/gc_test_norace.go

```diff
@@ -1,10 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
```

Comment:
Oh, missed this, can we just drop these changes from the first commit rather than committing them then removing them one commit later.

Reply:
Or specifically, because the second commit likely relies on the first commit to build, I recommend doing a commit that adds the `enableWithConfig(...)` func as a prerequisite for a second commit which does all the testing changes.